sdk/python/feast/pyspark/launcher.py

```diff
@@ -65,10 +65,26 @@ def _get_optional(option):
     )
 
 
+def _k8s_launcher(config: Config) -> JobLauncher:
+    from feast.pyspark.launchers import k8s
+
+    def _get_optional(option):
+        if config.exists(option):
+            return config.get(option)
+
+    return k8s.KubernetesJobLauncher(
+        namespace=config.get(opt.SPARK_K8S_NAMESPACE),
+        resource_template_path=_get_optional(opt.SPARK_K8S_JOB_TEMPLATE_PATH),
```

Comment:
Would it be better to use something like `config.get(option, default=None)`, instead of using _get_optional?

Reply:
fixed